criu/crtools.c

```diff
@@ -137,8 +139,11 @@ int main(int argc, char *argv[], char *envp[])
 		}
 
 		opts.exec_cmd = xmalloc((argc - optind) * sizeof(char *));
-		if (!opts.exec_cmd)
+		if (!opts.exec_cmd) {
+			pr_err("opts.exec_cmd failed to xmalloc.\n");
```

Comment:
The variable here is not important. Something like `Memory allocation failed` would be better in my opinion.